core-plugins/src/main/java/io/cdap/plugin/batch/aggregator/GroupByAggregator.java

```diff
@@ -124,6 +124,8 @@ public class GroupByAggregator extends RecordReducibleAggregator<AggregateResult
     put(GroupByConfig.Function.CONCATDISTINCT, "STRING_AGG(DISTINCT CAST(%s AS STRING) , \", \")");
     put(GroupByConfig.Function.LOGICALAND, "COALESCE(LOGICAL_AND(%s), TRUE)");
     put(GroupByConfig.Function.LOGICALOR, "COALESCE(LOGICAL_OR(%s), FALSE)");
+    put(GroupByConfig.Function.SUMOFSQUARES, "SUM(POW(%s, 2))");
```

Comment:
We should use the ANSI SQL `POWER` function. POW is a BigQuery-only method name

Reply:
Correct.

Also, the Spark function returns 0 if the input is empty, so we need to wrap this in a conditional block:

`CASE WHEN COUNT(%s) > 0 THEN <this expression> ELSE 0 END`

See https://github.com/cdapio/hydrator-plugins/pull/1575/files#diff-f285497c578d095fc4c85b1968df3e4dc52220cf71458410b5e1a347081f531fR121 for an example